### PR TITLE
Add news about Benoît's presentation at ApacheCon

### PR DESCRIPTION
**Pull request by rouazana**

### CODE REVIEW

File: src/homepage/_posts/2019-11-19-apache-con-eu.markdown [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+layout: post
+title:  "Benoît Tellier presents James at ApacheCon Europe"
+date:   2019-11-19  16:16:30 +0200
+categories: james apachecon
+---
+
+Benoît had the pleasure to present James at the ApacheCon at Berlin: "Gaining control over emails with Apache James".
```
Comment (chibenwa) [inline src/homepage/_posts/2019-11-19-apache-con-eu.markdown]: Benoit, no hat on the i

**Comment by Arsnael** [comment]
```
[ef05a49925a40d228f0ae1474b91973e2963ebe2] [ERROR] Errors: 
[ef05a49925a40d228f0ae1474b91973e2963ebe2] [ERROR]   CassandraMailRepositoryTest.storingAndRemovingMessagesConcurrentlyShouldLeadToConsistentResult » Execution
[ef05a49925a40d228f0ae1474b91973e2963ebe2] [ERROR]   CassandraMailRepositoryTest.storingAndRemovingMessagesConcurrentlyShouldLeadToConsistentResult » Execution
[ef05a49925a40d228f0ae1474b91973e2963ebe2] [ERROR]   CassandraMailRepositoryTest.storingAndRemovingMessagesConcurrentlyShouldLeadToConsistentResult » Execution
[ef05a49925a40d228f0ae1474b91973e2963ebe2] [ERROR]   CassandraMailRepositoryTest.storingAndRemovingMessagesConcurrentlyShouldLeadToConsistentResult » Execution
[ef05a49925a40d228f0ae1474b91973e2963ebe2] [ERROR]   CassandraMailRepositoryTest.storingAndRemovingMessagesConcurrentlyShouldLeadToConsistentResult » Execution
[ef05a49925a40d228f0ae1474b91973e2963ebe2] [ERROR]   CassandraMailRepositoryTest.storingAndRemovingMessagesConcurrentlyShouldLeadToConsistentResult » Execution
```
test this please

File: src/homepage/_posts/2019-11-19-apache-con-eu.markdown [hunk 2]
```diff
@@ -0,0 +1,13 @@
+---
+layout: post
+title:  "Benoît Tellier presents James at ApacheCon Europe"
```
Comment (trantienduchn) [inline src/homepage/_posts/2019-11-19-apache-con-eu.markdown]: > Benoit, no hat on the i

idem?

**Comment by chibenwa** [comment]
Merged

**Pull request closed without merging**